tests/e2e/e2e_setup_test.go

```diff
@@ -128,14 +128,9 @@ var (
 type IntegrationTestSuite struct {
 	suite.Suite
 
-	tmpDirs        []string
-	chainConfigs   []*chainConfig
-	dkrPool        *dockertest.Pool
-	dkrNet         *dockertest.Network
-	hermesResource *dockertest.Resource
-	initResource   *dockertest.Resource
-	valResources   map[string][]*dockertest.Resource
-	dockerImages   dockerconfig.ImageConfig
+	tmpDirs          []string
+	chainConfigs     []*chainConfig
+	containerManager *containers.Manager
```

Comment:
Should we make this an embedded field, like suite.Suite ?

Reply:
This is going to be moved further in incoming PRs from https://github.com/osmosis-labs/osmosis/pull/1591